src/pages/index.js

```diff
@@ -75,22 +131,64 @@ addCardButton.addEventListener("click", () => {
   cardPopup.open();
 });
 
-// CLASSES
-const profileEditPopup = new PopupWithForm(profileEditModal, handleFormSubmit);
+profileImageEditButton.addEventListener("click", () => {
+  profileImageEditPopup.open();
+})
+
+
+
+// ~~~~~~~~~~~~~~~~~ API ~~~~~~~~~~~~~~~~~ //
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "ba6f95f2-7e3a-4192-b9e4-9397fc618de9",
+    "Content-Type": "application/json"
+  }
+});
+
+let cardSection;
+
+api.getInitialCards()
```

Comment:
Could be better: Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all